html/captive-portal/templates/emails/emails-guest_sponsor_activation.html

```diff
@@ -80,6 +82,9 @@
       width: 50% !important;
       max-width: 50%;
     }
+
+  </style>
+  <style type="text/css">
```

Comment:
duplicated

Reply:
This is coming from mjml after processing.